Agents/SewageNetworkAgent/SewageNetworkAgent/src/main/java/uk/ac/cam/cares/jps/agent/sewage/SewerageNetworkAgent.java

```diff
@@ -0,0 +1,697 @@
+			String HG402 = HG_Instance[28];
+			String HG403 = HG_Instance[29];
+			String HG404 = HG_Instance[30];
+			String HG406 = HG_Instance[31];
+			String HG410 = HG_Instance[32];
+			String HG500 = HG_Instance[33];
+			String HG_GP002 = HG_Instance[34];
+			String HG_GP001A = HG_Instance[35];
+			String HG_GP001B = HG_Instance[36]; 
+			String HG_GP003A = HG_Instance[37];
+			String HG_GP003B = HG_Instance[38]; 
+			String HG_GP004A = HG_Instance[39];
+			String HG_GP004B = HG_Instance[40]; 
+			String HG_GP007A = HG_Instance[41];
+			String HG_GP007B = HG_Instance[42];
+			String HG_GP008A = HG_Instance[43];
+			String HG_GP008B = HG_Instance[44]; 
+			String HG_GP009A = HG_Instance[45]; 
+			String HG_GP009B = HG_Instance[46]; 
+			String HG_GP010A = HG_Instance[47]; 
+			String HG_GP010B = HG_Instance[48];
+
+
+			UpdateBuilder SewerageComponentHG_ub = 
+					new UpdateBuilder()
+					.addInsert(NodeFactory.createURI(KB + HG_Instance_Name), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(s4watr + "Pipe"))
+					.addInsert(NodeFactory.createURI(KB + HG_Instance_Name), NodeFactory.createURI(OS + "componentID"), HG001)		
+					.addInsert(NodeFactory.createURI(KB + HG_Instance_Name), NodeFactory.createURI(OS + "isAssociateWith"), NodeFactory.createURI(KB + "SewagePlant" + HG108))	
+					.addInsert(NodeFactory.createURI(KB + "SewagePlant" + HG108), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(OS + "SewagePlant"))		
+					.addInsert(NodeFactory.createURI(KB + HG_Instance_Name), NodeFactory.createURI(OS + "hasUsage"), NodeFactory.createURI(KB + "SewerageUsage" + HG302))
+					.addInsert(NodeFactory.createURI(KB + "SewerageUsage" + HG302), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(OS + "SewerageUsage"))		
+					.addInsert(NodeFactory.createURI(KB + HG_Instance_Name), NodeFactory.createURI(OS + "hasSewerageRecords"), NodeFactory.createURI(KB + "SewerageRecords" + HG_Instance_Name))
+					.addInsert(NodeFactory.createURI(KB + "SewerageRecords" + HG_Instance_Name), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(OS + "SewerageRecords"))
+
+
+					.addInsert(NodeFactory.createURI(KB + "SewerageRecords" + HG_Instance_Name), NodeFactory.createURI(OS + "hasOwnershipType"), NodeFactory.createURI(KB + "OwnershipType" + HG402))
+					.addInsert(NodeFactory.createURI(KB + "SewerageRecords" + HG_Instance_Name), NodeFactory.createURI(OS + "hasFunctionalState"), NodeFactory.createURI(KB + "FunctionalState" + HG401))
+					.addInsert(NodeFactory.createURI(KB + "OwnershipType" + HG402), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(OS + "OwnershipType"))
+					.addInsert(NodeFactory.createURI(KB + "FunctionalState" + HG401), NodeFactory.createURI(RDF_TYPE), NodeFactory.createURI(OS + "FunctionalState"));
+			UpdateRequest SewerageComponentHG_ur = SewerageComponentHG_ub.buildRequest();
+			AccessAgentCaller.updateStore("http://host.docker.internal:48888/ontosewage", SewerageComponentHG_ur.toString());
```

Comment:
Same comment as in PR #339 (https://github.com/cambridge-cares/TheWorldAvatar/pull/339#discussion_r1085418769): Please don't hard-code the endpoint URL here (and elsewhere below) - use a variable whose contents can be user-specified.

Reply:
Yes, I have done the same amendment as it the case of PR#339. Please check. Thank you.